gui/tsconfig.json

```diff
@@ -28,6 +28,7 @@
   ],
   "include": [
     "src/**/*.ts",
-    "src/**/*.tsx"
+    "src/**/*.tsx",
+    "assets/geo/*.json"
```

Comment:
FYI Had to mention JSON files because `gulp-typescript` couldn't see them. When using `tsc` direction that wasn't a problem. Either way...